internal/domain/file_content_response.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type FileContentResponse struct {
```

Comment:
Consider adding validation logic to ensure that `StartLine` is less than `EndLine`, as this can help prevent logical errors when processing file content.